Aws/Ec2/Client.py

```diff
@@ -28,3 +29,71 @@ def describe_regions(self) -> Optional[Region]:
         :return: Region object, or None if not found
         """
         return None
+
+    def run_instances(
+            self,
+            instance_type=None,
+            image_id=None,
+            user_data=None,
+            shutdown_behavior=None,
+            network_interfaces=None,
+            min_count=1,
+            max_count=1
+    ):
+        """
+        Start a new EC2 instance
+
+        :param instance_type: Optional Ec2 Instance type/size
+        :type instance_type: Optional[str]
+
+        :param image_id: Optional Instance AMI ID to run the instance with
```

Comment:
This is not optional

ImageId (string) -- The ID of the AMI. An AMI ID is required to launch an instance and must be specified here or in a launch template.